test/e2e/density.go

```diff
@@ -71,20 +70,6 @@ func printLatencies(latencies []podLatencyData, header string) {
 	Logf("perc50: %v, perc90: %v, perc99: %v", perc50, perc90, perc99)
 }
 
-// List nodes via gcloud. We don't rely on the apiserver because we really want the node ips
-// and sometimes the node controller is slow to populate them.
-func gcloudListNodes() {
```

Comment:
If we uploaded jenkins logs under hostname and not host ip this (or the apiserver list printing you're doing below) is not needed. The apiserver listing will be useless if the node controller takes > 5m  to populate host ip, which pre 1.0 was very often the case (much to my surprise). Eitherway removing it for greater justice is the right thing.